src/topology/vector_bundle.lean

```diff
@@ -0,0 +1,242 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri, Sebastien Gouezel.
```

Comment:
```suggestion
Authors: Nicolò Cavalleri, Sebastien Gouezel
```

Reply:
Okay, maybe we should undo this lint check. It was a waste of your time to have to fix this!